lib/rubocop/cop/lint/duplicate_branch.rb

```diff
@@ -83,21 +86,37 @@ module Lint
       #   else MEDIUM_SIZE
       #   end
       #
+      # @example IgnoreDuplicateElseBranches: true
```

Comment:
I think a better name would be `IgnoreDuplicateElseBranch`, as the `else` branch is just one.

Reply:
Agreed, changed.